src/server/database/Database/Implementation/CharacterDatabase.h

```diff
@@ -56,7 +56,6 @@ enum CharacterDatabaseStatements
     CHAR_SEL_CHAR_LEVEL,
     CHAR_SEL_CHAR_ZONE,
     CHAR_SEL_CHARACTER_NAME_DATA,
-    CHAR_SEL_CHARACTER_DATA_NAME,
```

Comment:
Oh. I see. I just did not see it before and it did not look like the file name was listed in your PR.